docs/developer-guide.md

```diff
@@ -37,6 +37,15 @@ To run happo tests:
 - create an `.env` file based on `env.example` and contact Button Inc. for credentials
 - `cd` into the desired package and run `yarn happo run`
 
+## Installing packages
+
+To install a new package
+
+- in the root directory, `yarn add [packageName] -W`
+- navigate to the root `package.json` and copy the package's line
+- navigate to the themes' `package.json` and paste the package's line
```

Comment:
Would using `yarn workspace "@button-inc/[theme]" add [packageName]`, yarn's built-in workspaces (which Lerna leverages) work here?

Reply:
I think you might be looking at an old commit; this turned out not to be necessary. But thanks for the info! I didn't know that about workspaces